test/support/test_complex.h

```diff
@@ -26,6 +26,24 @@
 #include <type_traits>
 #include <cassert>
 
+#if !_MSVC_STL_CPP17_COMPLEX_CONSTEXPR_FNC_BROKEN
+#    define STD_COMPLEX_TESTS_STATIC_ASSERT(...) static_assert(__VA_ARGS__)
+#else
+template <typename T>
+void
+__complex_tests_assert(T condition)
+{
+    assert(condition);
+}
+template <typename T>
+void
+__complex_tests_assert(T condition, const char*)
```

Comment:
I would suggest to rewrite the function to function-like macro in this case to be able to see more informative message from `assert` if something is wrong.

Reply:
Fixed.